app/assets/javascripts/osem-revisionhistory.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function() {
+    $('.show-changeset').click(function(){
+        if ($(this).text() == 'View Changes'){
+            $(this).text('Hide Changes');
+        }else {
+            $(this).text('View Changes');
+        }
+        $('#changeset-' + this.id).toggle();
+    });
+});
```

Comment:
Please add an empty line at the end of the file (this is the styling we are following)